CHANGES.rst

```diff
@@ -1,3 +1,23 @@
+2.1.0 (2023-11-08)
+==================
+
+Bugfixes
+--------
+
+- Fixed an issue where it was not possible to pass the ca_cert_data keyword argument in a proxy context when making SSL
+  requests. Previously, attempting to pass ca_cert_data in a proxy context would result in an error.
+  This issue has been resolved, and users can now pass ca_cert_data when making SSL requests through a proxy context. (`#3065 <https://github.com/urllib3/urllib3/issues/3065>`__)
+- Fixed decoding Gzip-encoded responses which specified ``x-gzip`` content-encoding. (`#3174 <https://github.com/urllib3/urllib3/issues/3174>`__)
+
+
+Deprecations and Removals
```

Comment:
This section is all removals, let's also move it to the top above bugfixes.

Reply:
Good idea! Done